src/components/AddressSearch/index.js

```diff
@@ -164,6 +166,12 @@ const AddressSearch = (props) => {
             values.state = longStateName;
         }
 
+        // UK addresses return countries (e.g. England) in the state field (administrative_area_level_1)
+        // So we use a secondary field (administrative_area_level_2) as a fallback
+        if (country === 'GB') {
```

Comment:
@jjcoffee Let's add this to `CONST.COUNTRY`.

Reply:
Good call, updated!